src/hooks/services/hooks.service.ts

```diff
@@ -143,7 +144,7 @@ export class HooksService {
       redirect_uri: callbackUrl,
       market: clientConfig.market,
       locale: clientConfig.locale,
-      test: this.config.tink.test ?? false,
+      test: clientConfig.realDataTest ?? false,
```

Comment:
In order to have a "smooth" transition can you keep algo the global config ? like that:
```suggestion
      test: clientConfig.realDataTest ?? this.config.tink.test ?? false,
```